cli_tools/common/utils/logging/service/logger.go

```diff
@@ -237,6 +237,12 @@ func (l *Logger) runWithServerLogging(function func() (*daisy.Workflow, error),
 	return logExtension, err
 }
 
+func removeNewLinesFromMultilineError(s string) string {
+	// first line in a multi error line is of "Multiple errors" type and doesn't need a separator
```

Comment:
I'm not familiar with the "multiple errors" type. Is that a golang thing? Daisy?

Reply:
https://github.com/GoogleCloudPlatform/compute-image-tools/blob/master/daisy/error.go#L154